src/apps/loan-list/helpers.ts

```diff
@@ -1,6 +1,6 @@
 import get from "lodash.get";
 import dayjs from "dayjs";
-import { LoanV2 } from "../../core/fbs/model/loanV2";
+import { LoanV2 } from "../../core/fbs/model";
 
```

Comment:
why this change

Reply:
The IDE that i uses says that this is the correct way to import it...